src/main/kotlin/com/hedvig/rapio/apikeys/Partner.kt

```diff
@@ -24,5 +24,6 @@ enum class Partner(val role: Role, val setupPaymentLinkVariation: String? = null
     SAMBLA(role = Role.COMPARISON),
     PEACH(role = Role.COMPARISON),
     MMG(role = Role.COMPARISON),
-    MOVELLO(role = Role.COMPARISON)
+    MOVELLO(role = Role.COMPARISON),
+    HEDVIGMANUALUPSELL(role = Role.COMPARISON)
```

Comment:
Cross-sell*, also underscores for readability 🙏 
```suggestion
    HEDVIG_MANUAL_CROSS_SELL(role = Role.COMPARISON)
```